library/src/webapp/skin/neo-ox/tool.css

```diff
@@ -1019,9 +1019,10 @@ tr.selectedFound td {
     font-weight: normal;
 }
 .login {
-    border: 1px solid #dedede;

```

Comment:
It is used at two other places.I will make changes in the file and not in this css.